modules/programs/kakoune.nix

```diff
@@ -592,7 +598,8 @@ let
         ++ [ "# UI options" ]
         ++ optional (ui != null) "set-option global ui_options ${uiOptions}"
 
-        ++ [ "# Key mappings" ] ++ map keyMappingString keyMappings
+        ++ [ "# User modes" ] ++ userModeStrings ++ [ "# Key mappings" ]
+        ++ map keyMappingString keyMappings
```

Comment:
I kept the funky formatting here for the reason described in https://github.com/rycee/home-manager/pull/1162#issuecomment-614883104.